### PR TITLE
HAWQ-1496. Add DISCLAIMER, LICENSE and NOTICE files into PXF rpm.

### PR DESCRIPTION
**Pull request by edespino**
The files will be installed into /opt/pxf-${project.version} directory.

### CODE REVIEW

**Comment by edespino** [comment]
reviewers: @jiny2 @radarwave @huor

File: pxf/build.gradle [hunk 1]
```diff
@@ -339,6 +339,18 @@ project('pxf') {
...
+        from('../resources/META-INF/DISCLAIMER') {
+            into "/opt/pxf-${project.version}"
+        }
+
+        from('../resources/META-INF/LICENSE') {
+            into "/opt/pxf-${project.version}"
+        }
+
+        from('../resources/META-INF/NOTICE') {
+            into "..//opt/pxf-${project.version}"
```
Comment (denalex) [inline pxf/build.gradle]: why is this line different from the others above ?

**Comment by radarwave** [comment]
Seems this is only for pxf main rpm, how about the other rpms related to pxf?
pxf-hbase
pxf-hdfs
pxf-hive
pxf-jdbc
pxf-json
pxf-service

**Comment by edespino** [comment]
@denalex & @radarwave - The need is to only deliver the license files in a single PXF rpm so the files can be accessed easily on the local file system.. We will continue to deliver the files with all PXF rpms.

**Comment by radarwave** [comment]
Thanks @edespino 

So I'm good with it. +1

**Comment by jiny2** [comment]
Thank you ed! @edespino 

+1 Look good to me .

**Comment by edespino** [comment]
Thanks @denalex for the review! I have fixed the typo.

**Pull request merged**